src/screens/Home/index.tsx

```diff
@@ -1,18 +1,22 @@
 import React from 'react';
 import Button from '~/components/Button';
+import Input from '~/components/Input';
 import Text from '~/components/Text';
 import {Home} from './styles';
 
 const HomeScreen = () => {
   return (
```

Comment:
You can destructure home styles to avoid mistakes like confusing styled components within the HomeScreen.

```suggestion
const HomeScreen = () => {
  const { Container } = Home;
  return (
```

Reply:
solved in c3336cc7837d319e35c2f400b5057b8470ce6e81